configure

```diff
@@ -125,63 +137,48 @@ check c++ "$CXX" 1
 rm -f test test.cpp
 cat << EOF > test.cpp
 #include <iostream>
-
-int main()
-{
-  std::cout << "Hello World!" << std::endl;
-  return 0;
-}
+int main() { std::cout << "A!"; return 0; }
```

Comment:
no need to change this, use the old code, it's immediate to recognize it's just an hello world.